src/components/tree_view/tree_view.tsx

```diff
@@ -272,167 +273,96 @@ export class EuiTreeView extends Component<EuiTreeViewProps, EuiTreeViewState> {
-                                        ? 'arrowDown'
-                                        : 'arrowRight'
-                                    }
-                                  />
-                                ) : null}
-                                {node.icon && !node.useEmptyIcon ? (
-                                  <span className="euiTreeView__iconWrapper">
-                                    {this.isNodeOpen(node) &&
-                                    node.iconWhenExpanded
-                                      ? node.iconWhenExpanded
-                                      : node.icon}
-                                  </span>
-                                ) : null}
-                                {node.useEmptyIcon && !node.icon ? (
-                                  <span className="euiTreeView__iconPlaceholder" />
-                                ) : null}
-                                <span
-                                  ref={ref}
-                                  className="euiTreeView__nodeLabel"
-                                >
-                                  {node.label}
-                                </span>
-                              </button>
-                              <div
-                                id={wrappingId}
-                                onKeyDown={(event: React.KeyboardEvent) =>
-                                  this.onChildrenKeydown(event, index)
-                                }
-                              >
-                                {node.children && this.isNodeOpen(node) ? (
-                                  <EuiTreeView
-                                    items={node.children}
-                                    display={display}
-                                    showExpansionArrows={showExpansionArrows}
-                                    expandByDefault={
-                                      this.state.expandChildNodes
-                                    }
-                                    {...label}
-                                  />
-                                ) : null}
-                              </div>
-                            </li>
-                          </React.Fragment>
-                        );
-                      }}
-                    </EuiI18n>
-                  )}
-                </EuiInnerText>
-              );
-            })}
-          </ul>
-        </EuiText>
+            {(listNavigationInstructions: string) => (
+              <EuiScreenReaderOnly>
+                <p id={instructionsId}>{listNavigationInstructions}</p>
+              </EuiScreenReaderOnly>
+            )}
+          </EuiI18n>
+        )}
+        <ul
+          css={cssStyles}
+          className={classes}
+          id={!this.isNested ? this.state.treeID : undefined}
+          aria-describedby={!this.isNested ? instructionsId : undefined}
+          {...rest}
```

Comment:
I was re-testing with VoiceOver and removing the containing `div[role="group"]` left us with an unstructured list of items. Adding a `role="list"` to the UL really shaped up the experience and shouldn't affect NVDA or JAWS because they retain the intrinsic understanding of UL being a list.

```suggestion
          role="list"
          {...rest}
```

Reply:
Wait are you telling me VoiceOver doesn't automatically give `ul`s a `role="list"??? 😕 what role does it get by default then?

Reply:
Well, none actually. Apple decided to remove the native role semantics when CSS was used to set `list-style: none;`

https://www.scottohara.me/blog/2019/01/12/lists-and-safari.html

Reply:
omg

Reply:
https://github.com/elastic/eui/pull/7513/commits/ba295580eec3131c2aeb91224b3c0c81302c591b

This required adding a lint rule skip (😭) and IMO is confusing enough to deserve its own inline comment as well. Thanks so much for the extra info Trevor! Today I (sadly) learnt!

Reply:
Good call Cee. At some point Apple may fix this oversight and having the context inline will be helpful.